RevenueCatUI/Templates/Components/TemplateComponentsView.swift

```diff
@@ -0,0 +1,202 @@
+//
+//  File.swift
+//  
+//
+//  Created by Josh Holtz on 6/11/24.
+//
+
+import RevenueCat
+import SwiftUI
+// swiftlint:disable all
+
+#if PAYWALL_COMPONENTS
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+private class ComponentPaywallData: ObservableObject {
+    @Published var selectedPackage: TemplateViewConfiguration.Package
+
+    init(selectedPackage: TemplateViewConfiguration.Package) {
+        self._selectedPackage = .init(initialValue: selectedPackage)
+    }
+}
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+struct TemplateComponentsView: TemplateViewType {
+
+
+    @StateObject
```

Comment:
nitpick: line break after start and at end of declaration, and remove the extra line break here